packages/cog/src/cog/cutline.ts

```diff
@@ -207,4 +213,19 @@ export class Cutline {
         const geojson = (await FileOperator.create(path).readJson(path)) as FeatureCollection;
         return new Cutline(geojson);
     }
+
+    /**
+     * Assume a cutline default based on a provided imagery name
+     *
+     * @param imageryName the name of the dataset being cut, should contain a substring denoting dataset type; sentinel, urban, rural.
+     */
+    static defaultCutline(imageryName: string): CutlineItem {
+        const foundNames = Object.keys(CutlineMap)
```

Comment:
this could be simplified with `const found = Object.keys(CutlineMap).find(f => imageryName.toLowerCase().includes(f))`

if we really want to check how many we could change the `find` to a `filter`

Reply:
I did actually try .find() using ts-node but got this `error TS2339: Property 'find' does not exist on type 'string[]'`. Need to upgrade I assume

Reply:
Filter version of above works as before and more concise.